src/utils/command.rs

```diff
@@ -46,24 +47,57 @@ where
     A: Display,
     T: Write,
 {
-    /// Queue the given command for later execution.
+    /// Queues the given command for execution in the near future.
     ///
     /// Queued commands will be executed in the following cases:
-    /// - When you manually call `flush` on the given writer.
-    /// - When the buffer is to full, then the terminal will flush for you.
-    /// - Incase of `stdout` each line, because `stdout` is line buffered.
     ///
-    /// # Parameters
+    /// * When `flush` is called manually on the given type implementing `io::Write`.
+    /// * When the buffer is to full, then the terminal will `flush` for you.
```

Comment:
There's a typo, should be `too`. But what `too full` does mean? I wouldn't complicate it and would just say:

```rust
/// * The terminal will `flush` automatically if the buffer is full.
```